### ISSUE TITLE
which data have to be downloaded from nuScenes

### ISSUE THREAD

**Issue by RashoAli** [exiawsh/StreamPETR#128]
I have a question regarding "getting started" with "Data Preparation". 

It says, 'Download the from nuScenes.' Do you mean the full dataset (v1.0) -->  all the training blobs and testing blobs?"

**Comment by RashoAli** [comment]
@exiawsh thanky you for the very fast reply.
unfrtunatly it is still hard to follow where to extrackt the data to.
currently i am using the folowing struckter

./data/nuscenes/v1.0-trainval{01, 02 ,03, ...}_blobs 
./data/nuscenes/v1.0-test_blobs

i get the flowing error whene runinng the line: "python tools/create_data_nusc.py --root-path ./data/nuscenes --out-dir ./data/nuscenes --extra-tag nuscenes2d --version v1.0"

AssertionError: Database version not found: ./data/nuscenes/v1.0-trainval

can you pleas provied me with more deatels of:
1. where i should extrackt the data to
2.  should i creat any folder in ./data/nuscenes?

**Comment by exiawsh** [comment]
You should unzip and merge the blobs files.
StreamPETR
├── projects/
├── mmdetection3d/
├── tools/
├── configs/
├── ckpts/
├── data/
│   ├── nuscenes/
│   │   ├── maps/
│   │   ├── samples/
│   │   ├── sweeps/
│   │   ├── v1.0-test/
|   |   ├── v1.0-trainval/
|   |   ├── nuscenes2d_temporal_infos_train.pkl
|   |   ├── nuscenes2d_temporal_infos_val.pkl
